### PR TITLE
Add sig-api-machinery emails in sigs.yaml

### PR DESCRIPTION
**Pull request by sttts**
Added SIG API-Machinery leads' emails to sigs.yaml

cc @deads2k @jpbetz @fedebongio

### CODE REVIEW

**Comment by sttts** [comment]
/hold

for @fedebongio @deads2k @jpbetz to confirm their email.

File: sigs.yaml [hunk 1]
```diff
@@ -14,19 +14,24 @@ sigs:
...
+      email: deads@redhat.com
     - github: fedebongio
       name: Federico Bongiovanni
       company: Google
+      email: fbongiovanni@google.com
     tech_leads:
     - github: deads2k
       name: David Eads
       company: Red Hat
+      email: deads@redhat.com
```
Comment (deads2k) [inline sigs.yaml:26]: lgtm

**Review by justaugustus — APPROVED** [review]
/lgtm
/approve

**Pull request merged**